seed/ts-sdk/any-auth/generate-endpoint-metadata/src/auth/AnyAuthProvider.ts

```diff
@@ -1,6 +1,16 @@
 // This file was auto-generated by Fern from our API Definition.
 
 import type * as core from "../core/index.js";
+import type { BearerAuthProvider } from "./BearerAuthProvider.js";
+import type { HeaderAuthProvider } from "./HeaderAuthProvider.js";
+import type { OAuthAuthProvider } from "./OAuthAuthProvider.js";
+
+export namespace AnyAuthProvider {
+    export interface AuthOptions
+        extends BearerAuthProvider.AuthOptions,
+            HeaderAuthProvider.AuthOptions,
+            OAuthAuthProvider.AuthOptions {}
```

Comment:
export interface AuthOptions
    extends BearerAuthProvider.AuthOptions,
        HeaderAuthProvider.AuthOptions,
        OAuthAuthProvider.AuthOptions {}

            should be

export type AuthOptions = BearerAuthProvider.AuthOptions | 
        HeaderAuthProvider.AuthOptions | 
        OAuthAuthProvider.AuthOptions;